--- Getopt-ExPar-0.01/expar_test.pl.perlpath	Mon Nov  9 17:17:22 1998
+++ Getopt-ExPar-0.01/expar_test.pl	Wed May 12 00:32:33 1999
@@ -1,4 +1,4 @@
-#!/rwasics_apps/gnu/bin/perl -w
+#!/usr/bin/perl -w
 
 # $Id$
 
--- Getopt-ExPar-0.01/Reference_Parser.pm.perlpath	Mon Nov  9 11:11:59 1998
+++ Getopt-ExPar-0.01/Reference_Parser.pm	Wed May 12 00:32:33 1999
@@ -1,4 +1,4 @@
-#!/rwasics_apps/gnu/bin/perl -w
+#!/usr/bin/perl -w
 
 package Reference_Parser;
 
